mypy/checker.py

```diff
@@ -1240,7 +1241,10 @@ def check_func_def(
                     # have no good way of doing this.
                     #
                     # TODO: Find a way of working around this limitation
-                    if len(expanded) >= 2:
+                    #
+                    # We suppress reachability warnings for empty generators (return; yield), since there's
+                    # no way to promote a function into a generator except by adding an "unreachable" yield.
```

Comment:
```suggestion
                    # We also suppress reachability warnings for empty generator functions
                    # (return; yield), since the most idiomatic way to promote a function into a
                    # generator function is often to add an "unreachable" yield.
```

Reply:
I had "also" at first, but I wanted to be able to remove the first comment w/o touching the second.

Reply:
Meh, I think I'd personally prioritise a readable comment over a tiny bit more churn in `git blame` :p

Reply:
Ahh, easy. I've just made the "TODO remove me" part second. :P